.github/styles/Blog/BrandName.yml

```
@@ -0,0 +1,11 @@
+extends: substitution
+message: Do you mean '%s' instead of '%s'? Notice the non-breaking space
```

```suggestion
message: Use a non-breaking space for brand name ('%s' instead of '%s')
```

My first idea when seeing the error message was "Why do they ask me if I wand to replace this by the same thing"